### PR TITLE
PMM-5283 Truncate fingerprint

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-5283

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1761

### CODE REVIEW

File: agents/postgres/pgstatstatements/pgstatstatements.go [hunk 1]
```diff
@@ -239,13 +240,14 @@ func makeBuckets(current, prev map[int64]*pgStatStatementsExtended, l *logrus.En
 			currentPSS.Tables = extractTables(currentPSS.Query, l)
 		}
 
+		fingerprint, _ := truncate.Query(currentPSS.Query)
```
Comment (BupycHuk) [inline agents/postgres/pgstatstatements/pgstatstatements.go:243]: We do already truncate it in https://github.com/percona/pmm-agent/blob/PMM-5283-truncate-query-example-and-fingerprint/agents/postgres/pgstatstatements/stat_statements_cache.go#L128, so maybe we shouldn't do it there.

File: agents/postgres/pgstatmonitor/pgstatmonitor.go [hunk 2]
```diff
@@ -247,10 +248,11 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 				m.l.Debugf("Normal query: %s.", currentPSM)
 			}
 
+			fingerprint, _ := truncate.Query(currentPSM.Fingerprint)
```
Comment (BupycHuk) [inline agents/postgres/pgstatmonitor/pgstatmonitor.go:251]: We already truncate fingerprint in https://github.com/percona/pmm-agent/blob/PMM-5283-truncate-query-example-and-fingerprint/agents/postgres/pgstatmonitor/stat_monitor_cache.go#L169

File: agents/mysql/perfschema/perfschema.go [hunk 3]
```diff
@@ -301,11 +301,12 @@ func makeBuckets(current, prev map[string]*eventsStatementsSummaryByDigest, l *l
 		}
 
 		count := inc(currentESS.CountStar, prevESS.CountStar)
+		fingerprint, _ := truncate.Query(*currentESS.DigestText)
```
Comment (JiriCtvrtka) [inline agents/mysql/perfschema/perfschema.go:304]: Do we need check isTruncated in these cases where you using truncate.Query?

File: agents/postgres/pgstatstatements/pgstatstatements.go (hunk 1 above)
Comment (askomorokhov) [reply agents/postgres/pgstatstatements/pgstatstatements.go]: Fixed.

File: agents/postgres/pgstatmonitor/pgstatmonitor.go (hunk 2 above)
Comment (askomorokhov) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go]: Fixed.

File: agents/mysql/perfschema/perfschema.go (hunk 3 above)
Comment (askomorokhov) [reply agents/mysql/perfschema/perfschema.go]: Yes, good point - we can at least pass it to QAN for future use.

**Review by JiriCtvrtka — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**